aten/src/ATen/native/Convolution.cpp

```diff
@@ -618,8 +635,9 @@ at::Tensor _convolution(
     }
 #endif
   } else if (input.device().type() == c10::DeviceType::CPU || input.device().type() == c10::DeviceType::CUDA) {
-    // TH/native only covers CPU/CUDA implementation.
-    if (params.groups == 1) {
+    if (params.use_caffe2_depthwise(input, weight)) {
```

Comment:
Didn't want to run the risk of messing with is_depthwise above as that could have more far reaching consequences.